state/subnets.go

```diff
@@ -438,11 +451,14 @@ func (st *State) addSubnetOps(id string, args network.SubnetInfo) (subnetDoc, []
 	if !unique {
 		return subnetDoc{}, nil, errors.AlreadyExistsf("subnet %q", args.CIDR)
 	}
+
+	// Unless explicitly placed, new subnets go into the alpha space.
+	// TODO (manadart 2020-08-12): We should determine the model's configured
+	// default space and put it there instead.
```

Comment:
Should we print a warning or something so we don't forget about this?

Reply:
I have a ticket to do this, and also to address https://bugs.launchpad.net/bugs/1883164.
Hoping to get this in for 2.9 so we can keep ratcheting towards our pure space concept.